Dockerfile

```diff
@@ -22,13 +22,13 @@ RUN npm run sbom
 # workaround for libxmljs startup error
 FROM node:20-buster as libxmljs-builder
 WORKDIR /juice-shop
-RUN apt-get update && apt-get install -y build-essential python3
+RUN apt-get update && apt-get install -y --no-install-recommends build-essential=12.6 python3=3.7.3-1
 COPY --from=installer /juice-shop/node_modules ./node_modules
 RUN rm -rf node_modules/libxmljs/build && \
   cd node_modules/libxmljs && \
   npm run build
 
-FROM gcr.io/distroless/nodejs20-debian11
+FROM gcr.io/distroless/nodejs20-debian11:latest
```

Comment:
Seems all fine to me, i have one small question though. I see you've used specific versions for alpine:3.20 image and in other places. I assume the latest version is always updated and remains secure, but i'd like to know your opinion on why you didn't choose a specific version here. Thanks. You can merge the pr after response

Reply:
Hi, I try to check version in gcr.io, but it's hard for me to find versioning at current image and if you have any propose inform me please

Reply:
I find solution
```
docker pull gcr.io/distroless/nodejs20-debian11@sha256:4840b78eb94649eea400df451cbe210b24196991c91126d40292c00509b9e838

```